modules/claims_api/app/mailers/claims_api/submission_report_mailer.rb

```diff
@@ -63,5 +51,73 @@ def path
         'submission_report.html.erb'
       )
     end
+
+    def pact_act_submissions(pact_act_data)
+      pact_act_data = ClaimsApi::ClaimSubmission.where(created_at: @from..@to) if pact_act_data.nil?
+
+      add_monthly_pact_data(pact_act_data)
+    end
+
+    def disability_compensation_submissions(disability_compensation_count)
+      if disability_compensation_count.nil?
+        disability_compensation_count = ClaimsApi::AutoEstablishedClaim
+                                        .where(created_at: @from..@to)
+                                        .pluck(:id)
+                                        .uniq
+                                        .size
+      end
```

Comment:
@rockwellwindsor-va Shouldn't the instance variables (`@from` and `@to`) be (`@date_from` and `@date_to`) ?

Reply:
Yep, thanks for catching that, fixing now.